boards/artix7_100t.json

```diff
@@ -1,31 +1,41 @@
 {
   "build": {
     "extra_flags": "-D ARTIX7_100T",
-    "f_cpu": "1500000000L",
+    "f_cpu": "50000000L",
     "hwids": [
       [
-        "0x0403",
-        "0x6010"
+         "0x002a",
+         "0x15ba"
       ]
     ],
     "ldscript": "link.ld",
     "mabi": "lp64",
     "march": "rv64imac",
     "mcmodel": "medany",
-    "mcu": "e51"
+    "mcu": "C-Class"
   },
-  "debug": {
-    "onboard_tools": [
-      "ftdi"
-    ]
+   "debug": {
+    "tools": {
+      "shaktidbg": {
+        "server": {
+          "arguments": [
+            "-f", "~/.platformio/packages/framework-shakti-sdk/bsp/third_party/artix7_100t/ftdi.cfg"
+          ],
+          "executable": "~/.platformio/packages/tool-openocd/openocd",
+	  "package": "tool-openocd"
+        }
+
+      }
+    }
```

Comment:
Already implemented [here](https://github.com/platformio/platform-shakti/blob/develop/platform.py#L52-L53)

Reply:
You can add the path to your SVD files in the debug section, for example like this [one](https://github.com/platformio/platform-ststm32/blob/develop/boards/b96b_f446ve.json#L20)

Reply:
Done.